packages/manager/modules/netapp/src/dashboard/index/template.html

```diff
@@ -6,8 +6,58 @@
         >
             <oui-tile-definition
                 term="{{:: 'netapp_dashboard_information_name' | translate }}"
-                description="{{ $ctrl.storage.name }}"
             >
+                <oui-tile-description data-ng-if="!$ctrl.isEditingName">
+                    {{ $ctrl.storage.name }}
+                </oui-tile-description>
+                <oui-tile-actions data-ng-if="!$ctrl.isEditingName">
+                    <button
+                        type="button"
+                        class="oui-button oui-button_ghost"
+                        data-ng-click="$ctrl.toggleNameEdition()"
+                    >
+                        <span class="oui-icon oui-icon-pen"></span>
+                    </button>
+                </oui-tile-actions>
```

Comment:
Has this new way of editing the name been validated by UX.
I cannot find this way anywhere, and I think we should keep some consistency in the way we perform actions.